src/Kit/System/Watchdog/IWatchedEventLoop.h

```diff
@@ -0,0 +1,190 @@
+#ifndef KIT_SYSTEM_WATCHDOG_IWATCHEDEVENTLOOP_H_
+#define KIT_SYSTEM_WATCHDOG_IWATCHEDEVENTLOOP_H_
```

Comment:
This file needs to be moved (up one directory) to the `src/Kit/System` directory.  File are stored by namespaces - and namespaces map 1-to-1 with directory names
```suggestion
#ifndef KIT_SYSTEM_IWATCHEDEVENTLOOP_H_
#define KIT_SYSTEM_IWATCHEDEVENTLOOP_H_
```